File: app/src/main/java/com/pkfcooperparry/template/ui/form/page/tab/TabFragmentViewModel.java
package com.pkfcooperparry.template.ui.form.page.tab;


import android.content.Context;

import com.pkfcooperparry.template.data.DataManager;
import com.pkfcooperparry.template.ui.base.BaseViewModel;
import com.pkfcooperparry.template.utils.rx.SchedulerProvider;

public class TabFragmentViewModel extends BaseViewModel<TabFragmentViewInterface> {

    public TabFragmentViewModel(DataManager dataManager, SchedulerProvider schedulerProvider) {
        super(dataManager, schedulerProvider);

    }

    @Override
    public void setViewInterface(Context context, TabFragmentViewInterface navigator) {
        super.setViewInterface(context, navigator);
    }


}

File: app/src/main/java/com/pkfcooperparry/template/ui/form/page/tab/TabFragmentViewModel.kt
package com.pkfcooperparry.template.ui.form.page.tab


import android.content.Context

import com.pkfcooperparry.template.data.DataManager
import com.pkfcooperparry.template.ui.base.BaseViewModel
import com.pkfcooperparry.template.ui.login.LoginActivity
import com.pkfcooperparry.template.utils.rx.SchedulerProvider

class TabFragmentViewModel(dataManager: DataManager, schedulerProvider: SchedulerProvider) : BaseViewModel<TabFragmentViewInterface>(dataManager, schedulerProvider) {

    override fun setViewInterface(context: Context, navigator: TabFragmentViewInterface) {
        super.setViewInterface(context, navigator)
    }


}
